bin/background_subtraction/run_background_subtraction.py

```diff
@@ -478,12 +479,19 @@ def write_bg_info_to_config(
     return
 
 
-def main(data_dir: Path, pipeline_config_path: Path, cytokit_config_path: Path):
+def main(
+    data_dir: Path,
+    pipeline_config_path: Path,
+    cytokit_config_path: Path,
+    out_base_dir: Path = None,
+):
     """Input images are expected to be stack outputs of cytokit processing
     and have names R001_X001_Y001.tif
     """
-    out_dir = Path("/output/background_subtraction")
-    config_out_dir = Path("/output/config")
+    if out_base_dir is None:
```

Comment:
With the default output paths being `/output/...`, it seems like it would make sense to use `Path("/")` as the default value of `out_base_dir`, perhaps in the `argparse` definition -- then a default value isn't required in this function definition, and the following few lines can be
```python
out_dir = out_base_dir / "output/background_subtraction"
config_out_dir = out_base_dir / "output/config"
```

Or maybe a default `out_base_dir` of `Path("/output")` with
```python
out_dir = out_base_dir / "background_subtraction"
config_out_dir = out_base_dir / "config"
```

(What initially caught my eye was a type hint of `Path` with a default value of `None`; I was going to say "the type should be `Optional[Path]`, then saw a slightly cleaner way to express the desired behavior.)

Reply:
Makes sense; will do.